pkg/test/cel/httproute_test.go

```diff
@@ -1209,6 +1209,18 @@ func TestHTTPRouteRule(t *testing.T) {
 				},
 			},
 		},
+		{
```

Comment:
Can you add a valid test case here?

Reply:
Yep, done. Included a "multiple unset is ok" part as well